app/services/conversation_service.rb

```diff
@@ -0,0 +1,14 @@
+class ConversationService
+  def initialize conversation
+    @conversation = conversation
+  end
+
+  def isSeen
+    @message = Message.by_conversation @conversation.id
+    for message in @message
```

Comment:
dùng each em nhé

Reply:
co mà chỗ này 
dùng update_all được k em

Reply:
được anh ạ. Nếu vậy sẽ không phải dùng vòng for nữa